README.md

```diff
@@ -10,19 +10,20 @@ The Interop Test Runner aims to automatically generate an interop matrix by runn
 The Interop Runner is written in Python 3. You'll need to install the
 following softwares to run the interop test:
 
-- Python3 modules. Run the following command:
+* Python3 modules. Run the following command:
 
-```bash
-pip3 install -r requirements.txt
-```
+   ```bash
+   pip3 install -r requirements.txt
+   ```
 
-- [Docker](https://docs.docker.com/engine/install/) and [docker compose](https://docs.docker.com/compose/). 
+* [Docker](https://docs.docker.com/engine/install/) and [docker compose](https://docs.docker.com/compose/).
 
-- [Development version of Wireshark](https://www.wireshark.org/download.html) (version 3.4.2 or newer).
+* [Development version of Wireshark](https://www.wireshark.org/download.html) (version 3.5.0 or newer).
```

Comment:
```suggestion
* [Development version of Wireshark](https://www.wireshark.org/download.html) (version 4.5.0 or newer).
```

With the latest release (`v4.4.2`) the rebinding tests are passing, but the connection migration test shows "First server packet on new path [...] did not contain a PATH_CHALLENGE frame" errors.

Those errors are gone when using a version built from the `master` branch, which will eventually become `v4.5.0`.

Reply:
FYI, you can download pre-release builds from https://www.wireshark.org/download/, no need to build from source (on major platforms).